features/plugin-uninstall.feature

```diff
@@ -36,3 +36,13 @@ Feature: Uninstall a WordPress plugin
       Warning: The 'edit-flow' plugin could not be found.
       """
     And the return code should be 0
+
+  Scenario: Uninstall all installed plugin
+    When I run `wp plugin uninstall --all`
+    Then STDOUT should be:
+      """
+      Uninstalled and deleted 'akismet' plugin.
+      Uninstalled and deleted 'hello.php' plugin.
```

Comment:
This will change to `'hello'` - see below re using `check_optional_args_and_all()`.

Reply:
It's giving me error in testing if I make it `hello`.